manuals/fast_and_fold.md

```diff
@@ -0,0 +1,26 @@
+# Fast and fold!
+AlphaPulldown V2 (beta) now can speed up your modelling process, save your computing time and resources by making multiple input feature matrices have 
```

Comment:
Arent' "speed up your modelling process, save your computing time" the same thing?